test/unit/ngsiv2/examples/contextRequests/updateContextMultientityTimestampPlugin1.json

```diff
@@ -17,11 +15,11 @@
       "TimeInstant": {
         "type": "DateTime",
         "value": "2016-05-30T16:25:22.304Z"
-      }
+      },
+      "id": "ws4",
+      "type": "WeatherStation"
```

Comment:
Is this change in the order of the keys in the JSON needed to make the tests using this .json as expectation to work?

A bit surprising... as the key order in the JSON files shouldn't matter...

Reply:
It matters for the way of tests is checking that json fields, just if both strings are equivalent.

Reply:
NTC